packages/test/src/test-worker-heartbeats.ts

```diff
@@ -144,3 +144,29 @@ test('Activity gets cancelled if heartbeat fails', async (t) => {
   });
   t.deepEqual(heartbeatsSeen, []);
 });
+
+test('No heartbeat is emitted with rouge activity', async (t) => {
```

Comment:
```suggestion
test('No heartbeat is emitted with rogue activity', async (t) => {
```

Reply:
Thanks, just merged without this, I'll fix in another PR